web/src/containers/Trade/utils.js

```diff
@@ -108,10 +109,10 @@ export const tradeHistorySelector = createSelector(
 );
 
 export const marketPriceSelector = createSelector(
-	[tradeHistorySelector],
-	({ data: tradeHistory }) => {
+	[tradeHistorySelector, getChartClose],
+	({ data: tradeHistory }, chartCloseValue) => {
 		const marketPrice =
-			tradeHistory && tradeHistory.length > 0 ? tradeHistory[0].price : 1;
+			tradeHistory && tradeHistory.length > 0 ? tradeHistory[0].price : chartCloseValue;
```

Comment:
@jsmitrah  what happens if chartCloseValue is also not there (null). What would then be displayed?

Reply:
@abeikverdi I handled the empty value in orderbook component